docker-compose-dev.yaml

```diff
@@ -21,71 +21,87 @@ services:
       context: .
       dockerfile: ./server/Dockerfile-dev
       args:
-        - http_proxy
-        - https_proxy
-        - no_proxy
+      - http_proxy
```

Comment:
The changes in this file address linging issues and bring docker-compose-dev up to speed with docker-compose